File: examples/src/Presentations/Conversion/PdfImportExample.cs
using System.IO
using Aspose.slides.Export

/*
This example imports a PDF document into Presentation. 
A new SlideCollection.AddFromPdf method creates slides from the PDF document 
and adds them to the end of the collection
*/


namespace Aspose.slides.Examples.CSharp.Presentations.Conversion
{
    public class PdfImportExample
    {
        public static void Run()
        {
            dataDir = RunExamples.GetDataDir_Conversion()

            pdfFileName = Path.Combine(dataDir, "welcome-to-powerpoint.pdf")
            resultPath = Path.Combine(RunExamples.OutPath, "fromPdfDocument.pptx")

            with slides.Presentation() as pres:
            {
                pres.slides.AddFromPdf(pdfFileName)
                pres.save(resultPath, slides.export.SaveFormat.PPTX)
            }
        }
    }
}

File: examples/src/Presentations/Conversion/pdf_import_example.py
import aspose.slides as slides


def import_from_pdf(global_opts):
    with slides.Presentation() as pres:
        pres.slides.add_from_pdf(global_opts.data_dir + "welcome-to-powerpoint.pdf")
        pres.save(global_opts.out_dir + "import_from_pdf_out.pptx", slides.export.SaveFormat.PPTX)
